src/main/java/strategy/operation/SupplyOperationHandler.java

```diff
@@ -0,0 +1,14 @@
+package strategy.operation;
+
+import db.Storage;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class SupplyOperationHandler implements OperationHandler {
+
+    @Override
+    public void handle(FruitTransaction fruitTransaction) {
+        Integer oldSum = Storage.fruits.get(fruitTransaction.getFruit());
```

Comment:
Handle the case when the fruit is not present in the Storage.fruits map to avoid NullPointerException.